scripts/update_node.py

```diff
@@ -37,13 +38,13 @@
     urllib.request.urlretrieve(download_url, filename)
 
     if '-win-' in suffix:
-      subprocess.check_call(['unzip', '-q', filename])
+      subprocess.check_call(unzip_cmd() + [filename])
       dirname = os.path.splitext(os.path.basename(filename))[0]
       shutil.move(dirname, 'bin')
       os.mkdir(dirname)
       shutil.move('bin', dirname)
       os.remove(filename)
-      subprocess.check_call(['zip', '-rq', filename, dirname])
+      subprocess.check_call(zip_cmd() + [filename, dirname])
```

Comment:
Since this is only ever run once every years or two a developer's machine (normally my machine) I think its reasonable to have `zip` in the path.  

I'd rather not split emsdk.py into multiple files for this reason.

Reply:
Also, I wonder if we should just update emsdk so that it doesn't require this modification to the windows node archive?

Reply:
I vaguely recall that is how it was originally, but someone reported an issue that windows doesn't have bin/ but linux and macos have. I prefer the consistency, so that the tools are in the same directory and can be run with `bin/node%.exe%` without needing to write platform-specific paths for each.

Reply:
> I think its reasonable to have `zip` in the path.

The fix is to be able to run this tool on Windows with 7z. I don't know where to obtain zip on Windows, and we've only used 7z.exe ever. If it's good for update_python.py, it should be good for update_node.py as well?

> I'd rather not split emsdk.py into multiple files for this reason.

These are not used in emsdk.py, but just in these two python files in scripts/ . I needed to be able to run update_node.py to develop Windows on ARM support.